### PR TITLE
Updating release instruction for the Libraries BOM

### PR DESCRIPTION
**Pull request by suztomo**
@meltsufin Thank you for feedback. Here is quick enhancements for the release instruction.

### CODE REVIEW

File: boms/cloud-oss-bom/RELEASING.md [hunk 1]
```diff
@@ -71,13 +88,15 @@ run `git config credential.helper store`.
 
 ### Build the release binary with Rapid (Legacy web UI)
```
Comment (chanseokoh) [inline boms/cloud-oss-bom/RELEASING.md:89]: How about making it clear that this section is not part of the current release process?

```suggestion
### (Depreacated) Build the release binary with Rapid (Legacy web UI)
```

Comment (chanseokoh) [reply boms/cloud-oss-bom/RELEASING.md]: Or maybe we should just remove this section. There do not exists instructions on the internal team page for BOM releasing. I remember the internal page used to have some relevant instructions, but they are now out of sync too much.

Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md]: That's nice idea. Removed.

**Comment by suztomo** [comment]
@meltsufin This is the enhancement for the release instruction.

File: boms/cloud-oss-bom/RELEASING.md [hunk 2]
```diff
@@ -89,24 +101,29 @@ new release is available on Maven Central.
       (no PR required)
     * https://github.com/googleapis/google-http-java-client/blob/master/docs/setup.md
     * https://github.com/googleapis/google-cloud-java/blob/master/TROUBLESHOOTING.md
+* In google3 run:
+    * `$ scripts/update_docs.sh <old version> <new version>`
+      * For example, `$ scripts/update_docs.sh 16.3.0 16.4.0`
+      * When asked whether to add changes to the first CL, answer "y".
+    * Sanity check the CL and send it for review. Use "Suggest reviewers" to pick a reviewer.
```
Comment (meltsufin) [inline boms/cloud-oss-bom/RELEASING.md:108]: "Suggest reviewers" was failing for me. Maybe say here who should be contacted?

**Review by meltsufin — APPROVED** [review]
This looks fine. But I think these instructions should move to an internal site.

File: boms/cloud-oss-bom/RELEASING.md [hunk 3]
```diff
@@ -5,21 +5,31 @@
...
+  "_linux_amd64.tar.gz". Place the `gh` executable somewhere in the `PATH` environment variable.
 
     * Run `gh auth login` to register your desktop with github.
 
 * Clone this repository onto your corp desktop, Ubiquity instance, or CloudTop. Do not use a laptop or personal machine as the release requires google3 access.
 
 * Install and configure the [repo tool](https://github.com/googleapis/github-repo-automation).
-
-* Ensure you have `p4` and `blaze` commands.
+  This updates code samples across ~100 repositories. You need `config.yaml` when you run it:
```
[the remark this answers is not in the document]
Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md:21]: It's the current working directory when running the command.

File: boms/cloud-oss-bom/RELEASING.md (hunk 2 above)
Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md:108]: "Suggest reviewers" not working is strange. But I'll add the account names after moving to an internal site.

**Pull request merged**